src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -1211,4 +1244,14 @@ private boolean needNumFilteredGenotypes(){
                 maxFractionFilteredGenotypes != MAX_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE ||
                 minFractionFilteredGenotypes != MIN_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE;
     }
+
+    /**
+     * Need the number of no-call genotyped samples?
```

Comment:
ok, i fixed the comment in needNumFilteredGenotypes the same way: "Should the number of filtered genotypes be considered for filtering?" , though it sounds a bit weird